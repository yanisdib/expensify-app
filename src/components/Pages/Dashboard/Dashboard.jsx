import React from 'react';
import ConnectedExpenseList from '../Expenses/ExpenseList';
import ExpenseListFilters from '../Expenses/ExpenseListFilters';

const Dashboard = () => (
    <div>
        <ExpenseListFilters />
        <ConnectedExpenseList />
    </div>
);

export default Dashboard;