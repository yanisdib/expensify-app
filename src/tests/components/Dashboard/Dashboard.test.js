import React from 'react';
import { shallow } from 'enzyme';
import Dashboard from '../../../components/Pages/Dashboard/Dashboard';

test('should generate Dashboard correctly', () => {
    const wrapper = shallow(<Dashboard />);
    expect(wrapper).toMatchSnapshot();
});