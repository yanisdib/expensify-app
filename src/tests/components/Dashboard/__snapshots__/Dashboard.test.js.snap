// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should generate Dashboard correctly 1`] = `
<div>
  <React.Memo />
  <React.Memo />
</div>
`;
